### PR TITLE
Add October 2023 newsletter

### PR DESCRIPTION
**Pull request by CollierCZ**
----
:books: Documentation preview :books:: https://writethedocs-www--2037.org.readthedocs.build/

### CODE REVIEW

File: docs/blog/newsletter-october-2023.rst [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+Write the Docs Newsletter – October 2023
+#########################################
+
+Ave, documentarians. Halloween may be approaching, but there's nothing scary about this month's batch of stories from the world of Write the Docs.
+
+The Write the Docs `Documentarian Salary Survey for 2023 <https://salary-survey.writethedocs.org/>`__ is open and accepting submissions! This is the survey's fifth year and, in addition to our regular salary and job-related questions, we're exploring our community's thoughts on "back to the office" mandates, feelings about job security in a volatile market, and attitudes towards pay transparency.
+
+So whether you're an employee, contractor, freelancer, or currently unemployed, `head on over <https://salary-survey.writethedocs.org/>`_ and help us make this year's survey the most useful yet.
+
+In conference news, `tickets for the in-person Australia conference </conf/australia/2023/tickets/>`__ are still open. If you'd like to apply for an `opportunity grant </conf/australia/2023/opportunity-grants/>`__, you have until 15 October.
```
Comment (plaindocs) [inline docs/blog/newsletter-october-2023.rst]: Speakers should be announced soon, right @swapnilogale ?

File: docs/blog/newsletter-october-2023.rst [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+If you put links within the content, `one reference suggested <https://readabilityguidelines.co.uk/content-design/links/#2-avoid-mid-sentence-links>`__ always putting the link at the end of the sentence to minimize distraction. Accessibility was an additional concern, but exact solutions were unclear because interpretation depends upon the screen reader. 
+
+--------------------------------
+Personality in Technical Writing
+--------------------------------
+
+There are still different views among documentarians on the issue of the importance of personality, customization, and tone in technical writing. People recently shared their thoughts on the topic, giving different points of view on whether such things belong in technical documents.
```
Comment (plaindocs) [inline docs/blog/newsletter-october-2023.rst]:
```suggestion
There are still different views among documentarians on the issue of the importance of personality, customization, and tone in technical writing.
```

harsh, but...

File: docs/blog/newsletter-october-2023.rst [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+People noted some potential benefits, including increased accountability for changes with PRs. Some also felt safer knowing any mistakes weren't theirs alone and that the CI checks and reviews helped keep entire pages from breaking. Someone also noted that with docs in the same repository as code, it was easier to enforce docs changes when code is updated, including potentially automating some changes.
+
+People had some suggestions to help overcome some roadblocks. Opening PRs early in the process lets reviewers check out changes locally to see the effects. Someone also suggested using the principle of pair programming in pair reviews for easier collaboration: run the docs locally and walk through the changes on a video call. It was also suggested to keep PRs small and therefore manageable, but batch really small things like typos together if each PR takes too long. And encourage improvements to the CI to catch typos earlier and also only run when needed (not linting untouched code).
```
Comment (plaindocs) [inline docs/blog/newsletter-october-2023.rst]: Random passive for no reason here?

File: docs/organizer-guide/newsletter/newsletter-process.rst [hunk 4]
```diff
@@ -120,15 +120,15 @@ Once all the story drafts have come in, it's time to assemble, based on the outl
...
    ------------------
 
    - `TITLE <https://jobs.writethedocs.org/LINK>`__, COMPANY (LOCATION)
 
    *To apply for these jobs and more, visit the* `Write the Docs job board <https://jobs.writethedocs.org/>`_.
 
    ----------------
-   Events coming up
+   Events Coming Up
    ----------------
```
Comment (plaindocs) [inline docs/organizer-guide/newsletter/newsletter-process.rst:132]: Personal preference for the lower case here, but :man_shrugging:

**Review by plaindocs — COMMENTED** [review]
Looks fine to me.

File: docs/blog/newsletter-october-2023.rst (hunk 1 above)
Comment (swapnilogale) [reply docs/blog/newsletter-october-2023.rst]: Yes, just overnight if I can

File: docs/blog/newsletter-october-2023.rst (hunk 3 above)
Comment (CollierCZ) [reply docs/blog/newsletter-october-2023.rst]: I think the idea was to have variety in sentence intros, but can change it.

Comment (CollierCZ) [reply docs/blog/newsletter-october-2023.rst]:
```suggestion
People had some suggestions to help overcome some roadblocks. Opening PRs early in the process lets reviewers check out changes locally to see the effects. Someone also suggested using the principle of pair programming in pair reviews for easier collaboration: run the docs locally and walk through the changes on a video call. Another idea was keeping PRs small and therefore manageable, but batching really small things like typos together if each PR takes too long. And encouraging improvements to the CI to catch typos earlier and also only runing when needed (not linting untouched code).
```

File: docs/organizer-guide/newsletter/newsletter-process.rst (hunk 4 above)
Comment (CollierCZ) [reply docs/organizer-guide/newsletter/newsletter-process.rst:132]: Yeah, I get that. Mostly I just noticed a mixture creeping in and wanted to get away from that and picking either standard seemed better. It's also something of a 🇺🇸 vs. 🇬🇧 thing. 🤷

Comment (plaindocs) [reply docs/organizer-guide/newsletter/newsletter-process.rst:132]: For sure your call, and for sure consistency is better than a mix.

File: docs/blog/newsletter-october-2023.rst [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+
+If you put links within the content, `one reference suggested <https://readabilityguidelines.co.uk/content-design/links/#2-avoid-mid-sentence-links>`__ always putting the link at the end of the sentence to minimize distraction. Accessibility was an additional concern, but exact solutions were unclear because interpretation depends upon the screen reader. 
+
+--------------------------------
+Personality in Technical Writing
+--------------------------------
+
+There are still different views among documentarians on the issue of the importance of personality, customization, and tone in technical writing.
+
+Some people said that strict accuracy was the most important in technical writing, while others said that there was room for creativity and humor as long as precision wasn’t compromised. In particular, they said that adding a bit of personality can make such writing more interesting, especially if the readers can handle it.
```
Comment (hillaryfraley) [inline docs/blog/newsletter-october-2023.rst]: "the most important in technical writing" seems like it needs a noun, like "the most important consideration in technical writing"

**Comment by swapnilogale** [comment]
@CollierCZ any chance of holding this newsletter from going out until Monday? Just hoping to get the speakers announced in a day or two

File: docs/blog/newsletter-october-2023.rst [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+
+People noted some potential benefits, including increased accountability for changes with PRs. Some also felt safer knowing any mistakes weren't theirs alone and that the CI checks and reviews helped keep entire pages from breaking. Someone also noted that with docs in the same repository as code, it was easier to enforce docs changes when code is updated, including potentially automating some changes.
+
+People had some suggestions to help overcome some roadblocks. Opening PRs early in the process lets reviewers check out changes locally to see the effects. Someone also suggested using the principle of pair programming in pair reviews for easier collaboration: run the docs locally and walk through the changes on a video call. Another idea was keeping PRs small and therefore manageable, but batching really small things like typos together if each PR takes too long. And encouraging improvements to the CI to catch typos earlier and also only runing when needed (not linting untouched code).
```
Comment (hillaryfraley) [inline docs/blog/newsletter-october-2023.rst]:
```suggestion
People had some suggestions to help overcome some roadblocks. Opening PRs early in the process lets reviewers check out changes locally to see the effects. Someone also suggested using the principle of pair programming in pair reviews for easier collaboration: run the docs locally and walk through the changes on a video call. Another idea was keeping PRs small and therefore manageable, but batching really small things like typos together if each PR takes too long. And encouraging improvements to the CI to catch typos earlier and also only running when needed (not linting untouched code).
```

File: docs/blog/newsletter-october-2023.rst (hunk 5 above)
Comment (plaindocs) [reply docs/blog/newsletter-october-2023.rst]: @hillaryfraley  Good call! Alternatively "accuracy was most important in ..."

**Review by hillaryfraley — APPROVED** [review]

File: docs/blog/newsletter-october-2023.rst (hunk 6 above)
Comment (plaindocs) [reply docs/blog/newsletter-october-2023.rst]: Sad to see no rune emoji

**Comment by CollierCZ** [comment]
> @CollierCZ any chance of holding this newsletter from going out until Monday? Just hoping to get the speakers announced in a day or two

Sure thing.

File: docs/blog/newsletter-october-2023.rst (hunk 5 above)
Comment (CollierCZ) [reply docs/blog/newsletter-october-2023.rst]: I remember changing it to that and thinking, "Remember to come back and add a noun." 🤦 Thanks!

Comment (CollierCZ) [reply docs/blog/newsletter-october-2023.rst]:
```suggestion
Some people said that strict accuracy was the most important consideration in technical writing, while others said that there was room for creativity and humor as long as precision wasn’t compromised. In particular, they said that adding a bit of personality can make such writing more interesting, especially if the readers can handle it.
```

File: docs/blog/newsletter-october-2023.rst (hunk 6 above)
Comment (CollierCZ) [reply docs/blog/newsletter-october-2023.rst]: That's because it's already a part of Unicode (ᚠ,ᛃ,ᚬ): https://en.wikipedia.org/wiki/Runic_(Unicode_block)

**Comment by plaindocs** [comment]
@Aaron228 I've just merged the Australian speaker announcement, so as long as you can see the news post and list of speakers, it's good to go.

File: docs/blog/newsletter-october-2023.rst (hunk 1 above)
Comment (CollierCZ) [inline docs/blog/newsletter-october-2023.rst]:
```suggestion
In conference news, the in-person Australia conference has just `announced its speakers </conf/australia/2023/news/announcing-speakers/>. Check out the amazing talks you have to look forward to and then `grab a ticket </conf/australia/2023/tickets/>`__. If you'd like to apply for an `opportunity grant </conf/australia/2023/opportunity-grants/>`__, you have until 15 October.
```

File: docs/blog/newsletter-october-2023.rst [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+#########################################
+
+Ave, documentarians. Halloween may be approaching, but there's nothing scary about this month's batch of stories from the world of Write the Docs.
+
+The Write the Docs `Documentarian Salary Survey for 2023 <https://salary-survey.writethedocs.org/>`__ is open and accepting submissions! This is the survey's fifth year and, in addition to our regular salary and job-related questions, we're exploring our community's thoughts on "back to the office" mandates, feelings about job security in a volatile market, and attitudes towards pay transparency.
+
+So whether you're an employee, contractor, freelancer, or currently unemployed, `head on over <https://salary-survey.writethedocs.org/>`_ and help us make this year's survey the most useful yet.
+
+In conference news, the in-person Australia conference has just `announced its speakers </conf/australia/2023/news/announcing-speakers/>. Check out the amazing talks you have to look forward to and then `grab a ticket </conf/australia/2023/tickets/>`__. If you'd like to apply for an `opportunity grant </conf/australia/2023/opportunity-grants/>`__, you have until 15 October.
```
Comment (CollierCZ) [inline docs/blog/newsletter-october-2023.rst]:
```suggestion
In conference news, the in-person Australia conference has just `announced its speakers </conf/australia/2023/news/announcing-speakers/>`__. Check out the amazing talks you have to look forward to and then `grab a ticket </conf/australia/2023/tickets/>`__. If you'd like to apply for an `opportunity grant </conf/australia/2023/opportunity-grants/>`__, you have until 15 October.
```

**Pull request merged**